src/navigation/examples/vertical-navigation-basic.js

```diff
@@ -392,9 +394,19 @@
            href: "#/adipscing"
         },
         {
-           title: "Lorem",
-           iconClass: "fa fa-gamepad",
-           href: "#/lorem"
+           title: "Help",
+           iconClass: "fa pficon-help",
+           href: "#/help",
+           mobileOnly: "true"
+        },
+        {
+           title: "User",
+           iconClass: "fa pficon-user",
+           mobileOnly: "true",
+           children: [
+              { title: "Preferences" },
```

Comment:
Can we change `Preferences` to `User Preferences` both here and in the header area? It was a UX request in the patternfly-react repo, so might as well update here as well.